build-site.sh

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
 # Define which models to embed in the site
-export ALLOY_MODELS="purealloy-modeling purealloy-modeling-samples"
+export LEGEND_MODELS="legend-modeling legend-modeling-samples"
```

Comment:
Renaming the names of the modules would break the logic. I suggest doing

`export LEGEND_MODELS=""`, to disable the feature for now (no content is being imported anyway)

Reply:
Done @maoo. See proposed change in https://github.com/brooklynrob/alloy/blob/master/build-site.sh